src/main/java/com/kenyajug/regression/application/ApplicationManager.java

```diff
@@ -0,0 +1,5 @@
+package com.kenyajug.regression.application;
+
+public class ApplicationManager {
```

Comment:
There is probably no use having empty classes, I prefer you deleted this

Reply:
You can include the diagram of the new structure in the project readme. I think it can help with the empty directories and handle cases where we introduce other api's in future